client/app/private-database/database/add/private-database-database-add.controller.js

```diff
@@ -120,5 +140,9 @@ angular.module("App").controller(
         getGrantLabel (grant) {
             return grant ? this.$scope.tr(`privateDatabase_add_bdd_new_user_grant_${grant}`) : "";
         }
+
+        isIpValid (ip) {
+            return this.validator.isValidIpv4Block(ip) || this.validator.isValidIpv4(ip);
```

Comment:
Isn't `this.validator.isValidIpv4(ip)` enough ? If I remember correctly, it accepts CIDRs